causing/examples/__main__.py

```diff
@@ -49,12 +47,14 @@
 )
 
 # Print json output
-with open(output_dir / "graphs.json", "w") as f:
-    json.dump(graphs, f, sort_keys=True, indent=4)
+dump_json(round_sig_recursive(graphs, 6), output_dir / "graphs.json")
+estimate_dat = estimate.filter_important_keys(estimate_dat)
```

Comment:
So are there other data from estimate_dat we do not need? Do we just need the filtered ones?

Reply:
There's also `biases`, `biases_std`, `cov_direct_hat`, `direct_hat`, `hessian_hat` and `sse_hat`. I was unsure whether they can be discarded generally, so I only filtered them before saving the JSON output.

Reply:
You're right we need them when evaluting the estimation results to potentially adapt the model.